test/csuite/incr_backup/main.c

```diff
@@ -0,0 +1,846 @@
+{
+    __wt_qsort(active->names, active->count, sizeof(char *), active_files_sort_function);
+}
+
+/*
+ * active_files_remove_missing --
+ *     Files in the previous list that are missing from the current list are removed.
+ */
+static void
+active_files_remove_missing(ACTIVE_FILES *prev, ACTIVE_FILES *cur, const char *dirname)
+{
+    uint32_t curpos, prevpos;
+    int cmp;
+    char filename[1024];
+
+    active_files_print(prev, "computing removals: previous list of active files");
+    active_files_print(cur, "computing removals: current list of active files");
+    curpos = 0;
+    /*
+     * Walk through the two lists looking for non-matches.
+     */
+    for (prevpos = 0; prevpos < prev->count; prevpos++) {
+again:
+        if (curpos >= cur->count)
+            cmp = -1; /* There are extra entries at the end of the prev list */
+        else
+            cmp = strcmp(prev->names[prevpos], cur->names[curpos]);
+
+        if (cmp == 0)
+            curpos++;
+        else if (cmp < 0) {
+            /*
+             * There is something in the prev list not in the current list. Remove it, and continue
+             * - don't advance the current list.
+             */
+            testutil_check(
+              __wt_snprintf(filename, sizeof(filename), "%s/%s", dirname, prev->names[prevpos]));
+            VERBOSE(3, "Removing file from backup: %s\n", filename);
+            remove(filename);
+        } else {
+            /*
+             * There is something in the current list not in the prev list. Walk past it in the
+             * current list and try again.
+             */
+            curpos++;
+            goto again;
+        }
+    }
+}
+
+/*
+ * active_files_free --
+ *     Free the list of active files.
+ */
+static void
+active_files_free(ACTIVE_FILES *active)
+{
+    uint32_t i;
+
+    for (i = 0; i < active->count; i++)
+        free(active->names[i]);
+    free(active->names);
+    active_files_init(active);
+}
+
+/*
+ * active_files_move --
+ *     Move an active file list to the destination list.
+ */
+static void
+active_files_move(ACTIVE_FILES *dest, ACTIVE_FILES *src)
+{
+    active_files_free(dest);
+    *dest = *src;
+    WT_CLEAR(*src);
+}
+
+/*
+ * table_updates --
+ *     Potentially make changes to a single table.
+ */
+static void
+table_updates(WT_SESSION *session, TABLE *table)
+{
+    WT_CURSOR *cur;
+    WT_ITEM item, item2;
+    WT_MODIFY modify_entries[MAX_MODIFY_ENTRIES];
+    OPERATION_TYPE op_type;
+    uint64_t change_count;
+    uint32_t i, nrecords;
+    int modify_count;
+    u_char *value, *value2;
+    char key[MAX_KEY_SIZE];
+
+    /*
+     * We change each table in use about half the time.
+     */
+    if (__wt_random(&table->rand) % 2 == 0) {
+        value = dcalloc(1, table->max_value_size);
+        value2 = dcalloc(1, table->max_value_size);
+        nrecords = __wt_random(&table->rand) % 1000;
+        VERBOSE(4, "inserting %d records into %s\n", (int)nrecords, table->name);
```

Comment:
We're not only inserting. So maybe "changing %d record in %s" or "operating on %d records in %s"?